gammapy/estimators/points/core.py

```diff
@@ -168,15 +179,26 @@ def write(self, filename, sed_type=None, format="gadf-sed", **kwargs):
             * "profile": Gammapy internal format to store energy dependent
                 flux profiles. Basically a generalisation of the "gadf" format, but
                 currently there is no detailed documentation available.
-        **kwargs : dict
-            Keyword arguments passed to `astropy.table.Table.write`.
+        overwrite : bool
+            Overwrite existing file`.
         """
+        filename = make_path(filename)
         if sed_type is None:
             sed_type = self.sed_type_init
-
-        filename = make_path(filename)
         table = self.to_table(sed_type=sed_type, format=format)
-        table.write(filename, **kwargs)
+
+        # TODO: rather ugly - better method?
+        if str(filename).split(".")[-1] != "fits":
```

Comment:
Maybe this: 
`if ".fits" in str(filename):`

Reply:
```
".fits" in Path(filename).suffixes
```